### PR TITLE
[hud] store rockset lambdas in version control

### PR DESCRIPTION
**Pull request by suo**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #226

This PR uses the Rockset CLI to store our lambdas locally. It also adds
a CI check to make sure our local stuff is in sync with Rockset.

### CODE REVIEW

File: torchci/README.md [hunk 1]
```diff
@@ -41,3 +41,18 @@ them to your local server.
...
+The source of truth for our query lambdas is in `rockset/`. We use the Rockset
+CLI to deploy these queries to Rockset. To edit a query:
+
+1. Follow the steps to [install and authenticate the Rockset
+CLI](https://www.npmjs.com/package/@rockset/cli#download--installation-instructions).
+2. Optionally, install the [Rockset VSCode
+extension](https://marketplace.visualstudio.com/items?itemName=RocksetInc.rockset-vscode).
+3. Edit your query lambda. The SQL is found in `rockset/<workspace>/__sql/`, and
+parameter definitions are found in `rockset/<workspace>`.
+4. You can test your query lambda using the [Rockset
```
Comment (atalman) [inline torchci/README.md:56]: From what I understand another way of testing would be to use rockset console to create and test new lambda. And once its working as copy/paste its content to `rockset/<workspace>/__sql/`

**Review by atalman — APPROVED** [review]
This is so great! This is exactly piece  where missing! I think we can also use [torchci/README.md] as goto documentation for Rockset.

File: torchci/README.md [hunk 2]
```diff
@@ -41,3 +41,18 @@ them to your local server.
 We use [Vercel](https://vercel.com/torchci) as our deployment platform. Pushes
 to `main` and any other branches will automatically be deployed to Vercel; check out
 the bot comments for how to view.
+
+## How to edit a Rockset query lambdas
```
Comment (janeyx99) [inline torchci/README.md:45]:
```suggestion
## How to edit Rockset query lambdas
```
nit

File: torchci/README.md [hunk 3]
```diff
@@ -41,3 +41,18 @@ them to your local server.
...
+CLI to deploy these queries to Rockset. To edit a query:
+
+1. Follow the steps to [install and authenticate the Rockset
+CLI](https://www.npmjs.com/package/@rockset/cli#download--installation-instructions).
+2. Optionally, install the [Rockset VSCode
+extension](https://marketplace.visualstudio.com/items?itemName=RocksetInc.rockset-vscode).
+3. Edit your query lambda. The SQL is found in `rockset/<workspace>/__sql/`, and
+parameter definitions are found in `rockset/<workspace>`.
+4. You can test your query lambda using the [Rockset
+CLI]https://www.npmjs.com/package/@rockset/cli/v/0.8.0#execute-and-test-query-lambda-sql).
```
Comment (janeyx99) [inline torchci/README.md:57]:
```suggestion
CLI](https://www.npmjs.com/package/@rockset/cli/v/0.8.0#execute-and-test-query-lambda-sql).
```

File: torchci/README.md [hunk 4]
```diff
@@ -41,3 +41,18 @@ them to your local server.
 We use [Vercel](https://vercel.com/torchci) as our deployment platform. Pushes
 to `main` and any other branches will automatically be deployed to Vercel; check out
 the bot comments for how to view.
+
+## How to edit a Rockset query lambdas
+
+The source of truth for our query lambdas is in `rockset/`. We use the Rockset
```
Comment (janeyx99) [inline torchci/README.md:47]: I see you included an empty rockset/README.md. 

Is that intended?

**Comment by janeyx99** [comment]
I do have a question. How does versioning work? Will these always be the latest versions?

File: torchci/README.md (hunk 4 above)
Comment (suo) [reply torchci/README.md:47]: oops

**Comment by suo** [comment]
> I do have a question. How does versioning work? Will these always be the latest versions?

Great question. I've added a check that the queries we use in the app are synced with the ones in source control.

**Pull request merged**